### ISSUE TITLE
About Finetuning MLLMs for image classification

### ISSUE THREAD

**Issue by JubSteven** [claws-lab/projection-in-MLLMs#2]
Thank you for the great work! There aren't much work focusing on the capabilities of large MLLMs on basic discriminative tasks like image classifications recently, and your work have provided valuable insights. 

I have two questions regarding the topic (regarding this [paper](https://yuhui-zh15.github.io/VLMClassifier-Website/), which also focuses on image classification using MLLM) .

- I wonder if any experiments are done using just a **finetuned** version of  CLIP on the datasets? I'm quite curious about the results of finetuned CLIP against finetuned MLLM.
- In the appendix of the paper mentioned above, it is pointed out that finetuning LLM and projection layer would cause training loss to be unstable. Have you encountered similar problems when finetuning LLaVA on the domain-shifted datasets?

**Comment by gaurav22verma** [comment]
Hi, thank you for your interest in our work! Glad that you found the insights valuable! As for the questions:
- We did not fine-tune the CLIP weights on any of the domain-specific datasets. That's primarily because the goal was to investigate the role of the projection layer, which doesn't really require tuning the input representations. Nonethless, we have included zero-shot CLIP results and also results for an MLP-based classifier on top of the CLIP representations, so that should provide a ballpark estimate of what CLIP fine-tuning numbers would look like.
- That's interesting! We did not encounter any such instability during training. Perhaps the point-of-difference wrt the paper you refer to is that they experiment with LoRA fine-tuning and that's what the training loss seems somewhat spiky. The training loss curve for projector tuning seems free of such spikes.   

Hope this helps!
